meta/autoid/autoid.go

```diff
@@ -129,6 +128,49 @@ type Allocator interface {
 	GetType() AllocatorType
 }
 
+// IDIterator represent a list of values allocated by Alloc.alloc().
+type IDIterator struct {
```

Comment:
maybe the computation in IDIterator should be under uint64.

refer the old code：`id := int64(uint64(min) + uint64(j)*uint64(increment))`

Reply:
What is the improvement to use `uint64` rather than `int64`? I mean the overflow problem still exists.